lib/xlibc/include/stdarg.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _XLIBC_STDARG_H
+#define _XLIBC_STDARG_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
nit: looks like a bug? In C++ we need cstdarg and it should have the stuff in `std::`?

Reply:
C++ assumes va_start, va_arg, va_copy and va_end are macros (which they are in this case) so they're global even when using cstdarg. Looking at libc++, all cstdarg does is include stdarg.h and import va_list into the std-namespace (so there's va_list and std::va_list).

Since we're using builtins here, we *might* not need to use `extern "C"` (I don't know how it relates to builtins and haven't experimented with it to find out), but I've seen musl use it with builtins in stdarg.h, too.

Reply:
Okay, ignore my comment then. We can review it when there's trouble.